docs/contributing/index.md

```diff
@@ -1,69 +1,94 @@
 
-!!!info
+If you believe that someone in the community has violated the code of conduct, please make a report by emailing conduct@autoware.org.
 
-    If you want to make changes to the guidelines in this documentation site, you can open a new discussion thread and propose it.
+## What Should I Know Before I Get Started?
 
-## Code of conduct
+### Autoware concepts
 
-We welcome contributions from everyone.  
-To ensure our community stays open and healthy, we adhere to the [Contributor Covenant](https://www.contributor-covenant.org/), a widely used [code of conduct](https://github.com/autowarefoundation/autoware/blob/main/CODE_OF_CONDUCT.md) adopted by many other communities listed [here](https://www.contributor-covenant.org/adopters/).
+To gain a high-level understanding of Autoware's architecture and design, the following pages provide a brief overview:
 
-Everyone participating in the Autoware community is expected to follow the code of conduct.  
-If someone in the community happens to be violating these terms, let the maintainers know, and we will address it as soon as possible.
+- [Autoware architecture](https://autowarefoundation.github.io/autoware-documentation/main/design/)
+- [Autoware concepts](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-concepts/)
 
-!!!warning
+For experienced developers, the [Autoware interfaces](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/) and [individual component pages](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/components/) should also be reviewed to understand the inputs and outputs for each component or module at a more detailed level.
 
-    Note that contributions that ignore guidelines are not contributions but nuisances.
+### Contributing to open source projects
 
-## Contribution workflows
+If you are new to open source projects, we recommend reading GitHub's [How to Contribute to Open Source guide](https://opensource.guide/how-to-contribute/) for an overview of why people contribute to open source projects, what it means to contribute and much more besides.
 
-There are several types of contributions and the workflows differ between the types.
-See the description for your contribution type.
+## How can I get help?
+
+Do not open issues for general support questions as we want to keep GitHub issues for confirmed bug reports. Instead, open a discussion in the Q&A category. For more details on our support mechanisms and links to troubleshooting guides, refer to our [Support guidelines](../support/index.md).
```

Comment:
our -> the

But `the support mechanisms` sounds a bit unclear? :thinking: 
Should we use `Autoware's` or something like that?

```suggestion
Do not open issues for general support questions as we want to keep GitHub issues for confirmed bug reports. Instead, open a discussion in the Q&A category. For more details on the support mechanisms and links to troubleshooting guides, refer to the [Support guidelines](../support/index.md).
```

Reply:
Is this better?

```suggestion
Do not open issues for general support questions as we want to keep GitHub issues for confirmed bug reports. Instead, open a discussion in the Q&A category. For more details on the support mechanisms for Autoware, refer to our [Support guidelines](../support/index.md).
```

Reply:
Yeah, it's nice. Are there any reasons that you used `our` for `our [Support guidelines]`?

Reply:
Oops. I updated the suggestion!

Reply:
Perfect!